src/input/mod.rs

```diff
@@ -26,7 +26,7 @@ struct State {
     mouse_pos_ndc: mint::Point2<f32>,
 }
 
-struct Delta {
+struct Diff {
```

Comment:
We've gone back to diff?

Reply:
Actually, we can keep it as `Delta`, but then we need to move/rename `Delta` trait. Not a problem, I'm just too lazy to do it and I wanted to check it asap